rules/checked-by-xxx/rule.md

```diff
@@ -54,14 +54,18 @@ Doing this will ensure:
 - You don’t forget anything important
 - The receiver knows who else agrees with the content
 
+::: tip
+Not everyone knows about this rule - make the "checked by" line a hyperlink to this rule to avoid confusion
```

Comment:
Do we want to mention this is mainly targeted at clients?  I don't think we need this for all internal check bys.